### PR TITLE
Add .twig file extension to templates

### PR DESCRIPTION
**Pull request by jonseg**
If we add .twig extension to templates, the IDEs (like PHPStorm) provides syntax highlighting, autocompletion, suggestions and more.

### CODE REVIEW

**Comment by lyrixx** [comment]
:+1: ; it eases IDE to detect the language.

File: src/controllers.php [hunk 1]
```diff
@@ -2,14 +2,11 @@
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpFoundation\JsonResponse;
-use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
```
Comment (fabpot) [inline src/controllers.php]: Should be reverted

**Comment by jonseg** [comment]
Last commit reverted.

**Comment by fabpot** [comment]
Thank you @jonseg.

**Pull request closed without merging**